src/main/scala/uk/gov/nationalarchives/downloadfiles/Lambda.scala

```diff
@@ -93,6 +96,8 @@ class Lambda {
             value("fileId", fileId),
             value("consignmentId", consignmentId)
           )
+          // add FileStatus with statusType as 'Upload' and value as 'Success'
```

Comment:
I don't think this comment is needed.

Reply:
Yeah, it's not required. I can remove it. Thanks.